packages/victory-core/src/victory-util/index.ts

```diff
@@ -0,0 +1,27 @@
+export { default as addEvents } from "./add-events";
+export * as Axis from "./axis";
+export * as Collection from "./collection";
+export * from "./common-props";
+export * as Data from "./data";
+export * as DefaultTransitions from "./default-transitions";
+export * as Domain from "./domain";
+export * as Events from "./events";
+export * as Helpers from "./helpers";
+export * as Hooks from "./hooks";
+export * as Immutable from "./immutable";
+export * as LabelHelpers from "./label-helpers";
+export * as LineHelpers from "./line-helpers";
+export * as Log from "./log";
+export * as PointPathHelpers from "./point-path-helpers";
+export * as PropTypes from "./prop-types";
+export * as Scale from "./scale";
+export * as Selection from "./selection";
+export type { SVGCoordinateType, SVGCoordinateBounds } from "./selection";
```

Comment:
When importing from `victory-core`, all types are exported at the top-level.  So in addition to `Selection.SVGCoordinateType`, we also must export these types manually.